examples/ImageStreamer/README.md

```diff
@@ -0,0 +1,232 @@
+# Ansible for HPE OneView and Image Streamer
+
+This repository contains a collection of sample ansible playbooks for using HPE Image Streamer to deploy, update and revert Golden Images to OS boot/run volumes for HPE Synergy compute module servers.
+
+## Requirements
+
+* [Ansible >= 2.0.2](http://docs.ansible.com/ansible/intro_installation.html)
+* Linux host (Ubuntu, CentOS, etc)
+* [HPE OneView Python SDK](https://github.com/HewlettPackard/python-hpOneView#installation)
+* Python >= 2.7.9
+* [Ansible modules for HPE OneView](https://github.hpe.com/Rainforest/oneview-ansible)
+
+If you are new to ansible, here's an interesting resource to get started: [Introduction to Ansible webinar  series](https://www.ansible.com/webinars-training/introduction-to-ansible)
+
+## Setup
+
+To run the ansible playbooks provided in this project, you should do the following:
+
+###	1. Clone the repository
+
+   ```git clone https://github.hpe.com/ImageStreamer/Ansible.git```
+
+###2. Configure the ANSIBLE_LIBRARY environmental variable
+
+   ```$ export ANSIBLE_LIBRARY=/path/to/oneview-ansible/library```
+
+###3. OneView Client Configuration
+
+   To use the Ansible OneView modules, you can store the configuration on a JSON file. This file is used to define the settings, which will be used on the OneView appliance connection, like hostname, username, and password. Here's an example:
+
+```json
+{
+  "ip": "172.25.105.12",
+  "credentials": {
+    "userName": "Administrator",
+    "authLoginDomain": "",
+    "password": "secret123"
+  },
+  "api_version": 200
+}
+```
+###Project structure
+The ansible playbooks are organized in folders to promote modularity and readability.
+
+```
+| - group_vars
+	| - group.yml
+  - host_vars
+     | - host1.yml
+      -  host2.yml
+      -  host3.yml
+  - tasks
+     | - task1.yml
+       - task2.yml
+       - task3.yml
+  - hosts
+    playbook1.yml
+    playbook2.yml
+    playbook3.yml
+  
+```
+
+**group_vars** - This folder contains al information for a group of hosts. In a typical server deployment scenario, the deployment network, server hardware and enclosure group would be same across similar hosts, the information can be captured and stored here. This can be referenced later as variables while writing playbooks.
+
+**host_vars** - This folder contains all host specific information.
+
+**tasks** - This folder may contain tasks which can be reused across different playbooks to promote modularity or independent tasks for each playbook
+
+**hosts** - It a text file to store inventory related information.
+
+**playbook<1..n>.yml** - This is entry point of ansible tasks, and this is where we tell ansible of what we want to achieve. Typical examples may include creation of server profile, powering on all server hardwares or updating firmware across all server hardwares.
+
+##Usage
+To run the playbooks, go the tasks folder
+```$ ansible-playbook -i /path/to/playbookdirectory/playbook.yml```
```

Comment:
If you need to go to the tasks folder before, shouldn't you provide a relative path instead of an absolute one?
Isn't `path/to/playbookdirectory/playbook.yml` instead of `/path/to/playbookdirectory/playbook.yml`?